src/components/ProductCard.css

```diff
@@ -26,12 +26,26 @@
 .bottle-image {
   position: absolute;
   left: -160px;
-  width: 370px;
   bottom: -5px;
+  width: 370px;
   transform: rotate(-15deg);
   z-index: 0;
 }
 
+.smaller-image {
```

Comment:
There might be a good reason for the absolute positioning and fixed widths, which i can not see in the small scope of PR without knowing the output in the browser ... But in most cases it's bad practice. If you can achieve the same results via css grid or flexbox and/or relative sizing, you should consider changing this in order to keep your layout responsive.

Reply:
I am still tinkering with the layout of the product card, but for now i am using position absolute as my elements are supposed to be placed quite randomly rather than all centered or all left etc. I am aware it's not the most responsively sustainable thing. I will keep it for now, but might change to flex/grid at some point down the line :)